### DISCUSSION TITLE
Nimble Commander defaults to Russian

### DISCUSSION THREAD

**Discussion by dertuxmalwieder** [mikekazakov/nimble-commander#352] [Q&A] [answered]
Curious: on a German macOS system, Nimble Commander 1.6.1 (MAS or website, same result) is in Russian. Is there a way to make it English, at least?

**Comment by mikekazakov** [comment, marked as answer]
Of course.
Nimble Commander doesn't choose the language to be shown in, that's up to MacOS to decide. If you ensure that English is placed as a second language in the system, MacOS should do the right thing automatically.
Alternatively, you can force a particular application to use a specific language: go to System Settings -> General -> Language & Region -> Applications.

**Reply by dertuxmalwieder** [reply]
That fixed it indeed. I added Russian as the "first second" language a while ago and didn't think about it much, but Nimble Commander seems to be the only application that does anything with this setting. Whoops, learned a thing.

Thank you a lot!

**Reply by mikekazakov** [reply]
You are welcome!
